src/main/webapp/WEB-INF/tags/instructor/results/expandCollapseButton.tag

```diff
@@ -0,0 +1,21 @@
+<%@ tag description="instructorFeedbackResultsTop - Expand Collapse Button" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%@ attribute name="sortType" required="true" %>
+<%@ attribute name="showAll" required="true" %>
+<div class="pull-right" style="margin-top:7px">
```

Comment:
The `pull-right` here is the root cause of the collision. For example, if `margin-top` is changed back to `50px`, the button will collide with the question panel. Since the `a` tag inside already has `pull-right`, this `div` does not need it. This `div` should instead stay as a block-level element to prevent collisions with other elements.